src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/CaseWorkerRefUsersController.java

```diff
@@ -233,6 +238,12 @@
          * This API will need to be revisited if it is to be used for business functionality.
          */
 
+        log.info("ENVIRONMENT NAME:::::: " + environmentName);
+
+        if (environmentName.equalsIgnoreCase("PROD")) {
```

Comment:
Can you please do the other way around like 
`"PROD". equalsIgnoreCase (environmentName)`? just to avoid NPE if by chance environmentName is null.